src/testUtils/mocks/entities.ts

```diff
@@ -10,6 +10,7 @@ import {
   CurrentAccount,
```

Comment:
All of this is already implemented in the `feat/MSDK-263-add-instruction` branch. Keep it for now, but if there are conflicts later, use what is already there